lib/elixir_analyzer/cli.ex

```diff
@@ -6,22 +6,19 @@ defmodule ElixirAnalyzer.CLI do
   @usage """
   Usage:
 
-    $ elixir_analyzer <exercise-name> <input path> <output path>  [options]
+    $ elixir_analyzer <exercise-name> <input path> <output path> [options]
 
   You may also pass the following options:
-    --skip-analysis                       flag skips running the static analysis
```

Comment:
I don't understand the purpose of skipping analysis except for debugging, so I got rid of that option too. I think it still exists somewhere in `feature` though, although it's never used.

Reply:
> I think it still exists somewhere in feature though, although it's never used.

Do you mean the ability to skip a single check? Because I don't see anything related to skipping analysis as a whole anywhere else 🤔